### PR TITLE
remove makefile, all hail `npm install`

### PR DESCRIPTION
**Pull request by 0atman**
# Details
Makefile is an unnecessary dependency. All dependency management is done by npm! To install npm, it's simply curl -L https://npmjs.com/install.sh | sudo sh`, as documented in README.

## QA
```
npm run clean
npm install
```

### CODE REVIEW

File: .install_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+set -e
+
+echo "Installing gulp deps..."
+type gulp || sudo npm install -g gulp
+type gem || sudo apt install ruby-dev || echo "gem missing - please isntall manually!" && type gem
+
+if ! type scss-lint 2> /dev/null || [[ $(scss-lint --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+") < 0.34 ]]; then
+	sudo gem install scss-lint
```
Comment (nottrobin) [inline .install_dependencies.sh]: could we remove the tab character? not necessary in .sh files

File: Makefile [hunk 2]
```diff
@@ -1,17 +0,0 @@
-install:
```
Comment (nottrobin) [inline Makefile]: :heart:

File: Makefile [hunk 3]
```diff
@@ -1,17 +0,0 @@
...
-	@echo "Installing missing gulp deps..."
-	@type npm || curl -L https://npmjs.com/install.sh | sudo sh
-	@type gulp || sudo npm install -g gulp
-	@type gem || sudo apt install ruby-dev || echo "gem missing - please isntall manually!" && type gem
```
Comment (deadlight) [inline Makefile]: Typo: "install"

File: Makefile [hunk 4]
```diff
@@ -1,17 +0,0 @@
-install:
-	@echo "Installing missing gulp deps..."
```
Comment (deadlight) [inline Makefile]: consider "dependencies", we're not paying by the character

File: README.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ To get started you'll need some stuff:
 - [Latest release](https://github.com/ubuntudesign/web-style-guide/archive/master.zip)
 - install [Ruby](https://www.ruby-lang.org/en/)
 - Install [Sass](http://sass-lang.com/install)
-- Install [Node.js](http://nodejs.org)
+- Install [Node.js](http://nodejs.org) (`curl -L https://npmjs.com/install.sh | sudo sh`)
```
Comment (nottrobin) [inline README.md]: We should document this slightly more clearly. I'll try to PR it.

File: .install_dependencies.sh [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+set -e
+
+echo "Installing gulp deps..."
```
Comment (deadlight) [inline .install_dependencies.sh]: "dependencies", we're not paying by the character

File: .install_dependencies.sh [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+type gulp || sudo npm install -g gulp
+type gem || sudo apt install ruby-dev || echo "gem missing - please isntall manually!" && type gem
```
Comment (deadlight) [inline .install_dependencies.sh]: Typo: "install"

**Pull request merged**